Landing page (#225)
unlock-protocol/docs: docs/getting-started/glossary.md

    <dd>Short for decentralized finance.</dd>

  <dt>DYOR</dt>
    <dd>Do your own research. Another way to say "don't take my word for it".</dd>

  <dt>Factory Contract</dt>
    <dd>A factory contract is a particular type of smart contract that creates other smart contracts.</dd>

  <dt>Key</dt>
    <dd>In the Unlock ecosystem, a “Key” is an NFT (minted by a Lock) that has 
    additional features added to it to facilitate the management of memberships, 
    such as an expiration date.</dd>
  
  <dt>Key Manager</dt>
    <dd>An assigned title set on an Unlock key which gives a specific wallet
    address authorization to transfer, share or cancel.</dd>

  <dt>Lock</dt>
    <dd>In the Unlock ecosystem, a “Lock” is a smart contract that creates 
    (or “mints”) NFTs. Locks also can be used to ensure “members-only” access 
    to resources like online content, live events, online certifications, or 
    other “members-only” experiences.</dd>
  
  <dt>Lock Manager</dt>
    <dd>An assigned <a href="/core-protocol/public-lock/access-control#roles">role </a>
    set on a Lock contract which gives the highest level of permissions to the 
    wallet address set to that role.</dd>
  

>   <dt>Locksmith</dt>
>     <dd>A community member or builder in the Unlock ecosystem.</dd>
> 

Reviewer: ```suggestion
  <dt>Locksmith</dt>
    <dd>The name of the API backend service built by the Unlock Labs team and which offers optional features such as metadata-hosting, credit card payments...  It is also how we sometimes call community member or builder in the Unlock ecosystem. </dd>

```

---
title: Glossary
sidebar_position: 2
description: >- 
  A glossary of terms used in the Unlock Protocol community and also
  the greater web3 ecosystem we belong to.
---

# Glossary

Welcome, Locksmith! Let’s start with getting the lay of the land with a few common terms you’ll see across the Unlock ecosystem.

<dl>
  <dt>Crypto Wallet</dt>
    <dd>A digital wallet to hold your currencies, collectibles (eg, NTFs), and 
    used to log into dApps (example: Metamask, Rainbow, Coinbase Wallet)</dd>

  <dt>DAO</dt>
    <dd>Acronym for Decentralized Autonomous Organization</dd>

  <dt>dApp</dt>
    <dd>Short for decentralized application.</dd>
            
  <dt>Dashboard</dt>
    <dd>A web app built by Unlock Labs for creating "Locks", managing "Keys", 
      and more.</dd>

  <dt>DeFi</dt>
    <dd>Short for decentralized finance.</dd>

  <dt>DYOR</dt>
    <dd>Do your own research. Another way to say "don't take my word for it".</dd>

  <dt>Factory Contract</dt>
    <dd>A factory contract is a particular type of smart contract that creates other smart contracts.</dd>

>   <dt>Fren</dt>
>     <dd>Short for friend.</dd>
> 
>   <dt>GM</dt>
>     <dd>Good morning, but can also mean good day, good night, good evening</dd>
> 
>   <dt>GN</dt>
>     <dd>Good night.</dd>
> 
>   <dt>Hodl</dt>
>     <dd>To hold but specifically a token asset, to not panic sell.</dd>
>   

Reviewer: ```suggestion
```

---
title: Glossary
sidebar_position: 2
description: >- 
  A glossary of terms used in the Unlock Protocol community and also
  the greater web3 ecosystem we belong to.

> ---

Reviewer: Let's add the following:
UDT, Key Manager, Referrer, Paywall (our application)

Author: Added these plus the ones from @ccarfi document in notion. Updated a few of the definitions that were already there to match his which I liked.

<dl>

>   <dt>anon</dt>

Reviewer: Not sure we need these :D Let's focus on the Unlock stuff?

Author: Removed

    <dd>anonymous</dd>

>   <dt>ape</dt>
>     <dd>verb; to invest heavily in an asset.</dd> 
>     <dd>noun; may refer to an NFT from BAYC (Bored Ape Yacht Club) or a derivative</dd>

Reviewer: ```suggestion
```

Author: Removed

>   <dt>Bitcoin</dt>
>     <dd>A cryptocurrency ($BTC) on the Bitcoin blockchain which is only a ledger
> and does not have smart contracts natively.</dd>
> 
>   <dt>blockchain</dt>
>     <dd>A shared immutable ledger that facilitates the process of recording transactions
> and tracking assets over a decentralized network.</dd> 

Reviewer: ```suggestion 
```

Author: Removed

  <dt>crypto wallet</dt>
    <dd>A digital wallet to hold your currencies, collectibles (eg, NTFs), and used to 
log into dApps (example: Metamask, Rainbow, Coinbase Wallet)</dd>

  <dt>DAO</dt>
    <dd>Acronym for Decentralized Autonomous Organization</dd>

  <dt>dApp</dt>
    <dd>Short for decentralized application.</dd>
            
  <dt>Dashboard</dt>
    <dd>A web app built by Unlock Labs for creating "Locks", managing "Keys", 
      and more.</dd>

  <dt>DeFi</dt>
    <dd>Short for decentralized finance.</dd>

>   <dt>degen</dt>
>     <dd>Degenerate, usually self-assigned, and in the crypto community often times
> warn with a badge of honor.</dd>

Reviewer: ```suggestion
```

Author: Removed

>   <dt>doxxed</dt>
>     <dd>not anonymous</dd>
> 
>   <dt>DYOR</dt>
>     <dd>Do your own research. Another way to say "don't take my word for it".</dd>
> 
>   <dt>fren</dt>
>     <dd>Short for friend.</dd>
> 
>   <dt>gm</dt>
>     <dd>Good morning, but can also mean good day, good night, good evening</dd>
> 
>   <dt>gn</dt>
>     <dd>Good night.</dd>
> 
>   <dt>hodl</dt>
>     <dd>To hold but specifically a token asset, to not panic sell.</dd>

Reviewer: ```suggestion
```

Author: Removed

  
  <dt>Key</dt>
    <dd>Membership NFT minted from a "Lock" contract.</dd>
  

>   <dt>LFG</dt>
>     <dd>Acronym Let’s F&%*ing Go</dd>

Reviewer: ```suggestion
```

  <dt>Lock</dt>
    <dd>Customized smart contract generated from our factory/deployer contract.</dd>

>   <dt>NGMI</dt>
>     <dd>Never going to make it.</dd>

Reviewer: ```suggestion
```

>   <dt>rug / rugged</dt>
>     <dd>To get the proverbial rug pulled out from beneath you. eg, in Twitter 
>     Spaces, when the space fails or someone’s mic isn’t working, they got rugged. 
>     Or, in an NFT project, when the team doesn’t deliver their promises and take 
>     the money and run, the collectors have been rugged.</dd>
> 
>   <dt>WAGMI</dt>
>     <dd>We're all going to make it.</dd>

Reviewer: ```suggestion
```
